modules/ROOT/pages/billing/how-to-pay.adoc

```diff
@@ -0,0 +1,156 @@
+
+Aura usage is metered and aggregated based on billing dimensions, the result is priced according to Aura pricing plans.
+
+== Pay as you go
+
+Get started by creating an Aura account, no sales team is needed.
+Cancel at any time, with no extra charge.
+Pay as you go has no expiration date, it continues as long as you use Aura.
+
+Usage is billed monthly for the previous month. At every billing cycle, usage from the previous month is priced, invoiced and then charged to the customer’s payment method.
+
+Get billed directly with Neo4j using a credit card as a payment method. 
+Or, get billed through GCP, AWS or Azure Marketplace where usage is reported hourly to the marketplace. At the marketplace’s billing cycle, Aura usage is aggregated and charged as part of the customer’s cloud bill.
+
+== Prepaid
+
+Commit to using Aura for a minimum of one year and buy Aura Consumption Units (ACUs).
+
+Signing up requires the sales team, and prepaid does not renew automatically. 
+
+There's two separate channels for purchasing ACUs:
+* Direct invoicing with Neo4j
+* Purchasing ACUs through marketplace private offers (GCP, AWS or Azure).
+
+There is no billing integration with marketplaces for prepaid contracts. 
+The contract is executed as a private offer, Neo4j instructs the marketplace to invoice the customer for a number of ACUs and nothing else. 
+Neo4j does not provision an Aura environment automatically in this case, does not report usage to the marketplace for these deals, and the marketplace does not keep track of the remaining balance.
+
+== Aura Consumption Unit (ACU)
+
+Each billing dimension has its own unit of measure, GB-hours, GB-minutes, MB-hours, GB etc.
+To aggregate and then pay for usage across different dimensions, Aura Consumption Units (ACUs) are used.
+All Aura resources are priced in ACUs.
+Customers purchase ACUs as part of their prepaid contract, and then use them to cover for the usage that is also expressed in ACU.
+
+* ACUs have a start date and an expiration date. ACUs need to be used before their expiration date, or they are forfeited.
+
+== Contract lifecycle
+
+. The customer signs a contract with Neo4j for a given period (one or more contract terms) and buys a number of ACUs to be used in each yearly contract term.
+. The customer uses Aura resources priced in ACUs.
+. The usage (priced in ACU) is deducted from the ACU balance throughout the month.
+. If the customer runs out of ACUs before their expiration date, they are be invoiced for the overage.
+. At expiration date, all ACUs remaining are forfeited. If the current term is part of a multi-year contract, some of the remaining balance can be rolled over to the next term, if one exists. This has to be documented in the initial contract.
+
+== Availability matrix
+
+[.tabbed-example]
+====
+
+[.include-with-AuraDB]
+=====
+
+Free, Professional, Business Critical, and Virtual Dedicated Cloud plans
```

Comment:
```suggestion
Free, Professional, Business Critical, and Virtual Dedicated Cloud tiers
```
I was under the impression that we use _tiers_ instead of _plan_?